pkgs/applications/science/electronics/hyp2mat/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, bison
+, flex
+, gengetopt
+, help2man
+, groff
+, libharu
+, autoreconfHook
+, pkgconfig
+, libpng
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hyp2mat";
+  version = "0.0.17";
+
+  src = fetchFromGitHub {
+    owner = "koendv";
+    repo = "hyp2mat";
+    rev = version;
```

```suggestion
    rev = "v${version}";
```

This doesn't build right now because you are missing the v in the tag name.

---

This appears to build fine for me without the v. For instance, if I use `nixpkgs-review rev HEAD` everything seems fine. Could there be a reason I'm falsely getting a successful build for this?

---

I've gone ahead and made the other change and pushed it up. I'm holding off on this one until I hear back from you. If you know it's the right thing to do, I'm happy to make the change. But, I am curious to know why my build still appears to work. Moreover, I've been [told to do the opposite](https://github.com/NixOS/nixpkgs/pull/92879#pullrequestreview-446830865) in the past. [Here](https://github.com/whitequark/libfx2/releases) are the releases for that library.

---

We don't put the v in the version string, but we need to make rev match the tag name exactly.

---

Great, that makes sense. Thanks for taking the time to explain!